### PR TITLE
Add changelog

### LINKED ISSUE #2: Add a change log
Post-populate with changes from add/balance-table

### PR DESCRIPTION
**Pull request by Lebbitt**
Wasn't too sure how to write it or split it up so I'm open to any and all feedback! 
Only added notes for the first pull request that was merged, will update when other pull requests are merged to main

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Release 0.1 (07/01/23)
+
+### Additions
```
Comment (g-duff) [inline Changelog.md]: Could we go with the same headings as [keep a changelog](https://keepachangelog.com/en/1.0.0/)?

File: Changelog.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Change parameters for separate out entities
+- Change git ignore
+- Change import ordering
+- Change looping over iterables
+- Change pie chart to be with main file
+- Change dictionary comprehension
+
+### Fixes
+- Fix test import
+- Fix failing test cases
```
Comment (g-duff) [inline Changelog.md]: No fixes, since we haven't released yet

File: Changelog.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+### Remove
+- Remove duplicate function
+- Remove main file
```
Comment (g-duff) [inline Changelog.md]: No removes yet for the same reason

File: Changelog.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+## Release 0.1 (07/01/23)
+
+### Additions
+- Add read, sort and print json data
+- Add tests for functions
+- Add ability to create pie chart
+- Add requirements
+- Add project file
+- Add python import file
+- Add examples to ignore
```
Comment (g-duff) [inline Changelog.md]: I would condense these to a "feature level" overview. What features did we add last time?

**Review by g-duff — CHANGES_REQUESTED** [review]

File: Changelog.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+## Release 0.1 (07/01/23)
```
Comment (g-duff) [inline Changelog.md]: We haven't released yet ;)

**Review by g-duff — CHANGES_REQUESTED** [review]

File: Changelog.md (hunk 2 above)
Comment (Lebbitt) [reply Changelog.md]: done

File: Changelog.md (hunk 3 above)
Comment (Lebbitt) [reply Changelog.md]: done

File: Changelog.md (hunk 1 above)
Comment (Lebbitt) [reply Changelog.md]: done

File: Changelog.md (hunk 5 above)
Comment (Lebbitt) [reply Changelog.md]: done

File: Changelog.md (hunk 4 above)
Comment (Lebbitt) [reply Changelog.md]: done

File: Changelog.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+## [Unreleased] (07/01/23)
+
+### Added
+- Read and sort json files
```
Comment (g-duff) [inline Changelog.md]: How do we sort them? Not against having this point though.

Comment (Lebbitt) [reply Changelog.md]: done

File: Changelog.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+## [Unreleased] (07/01/23)
```
Comment (g-duff) [inline Changelog.md]: This date looks like a pain to maintain every time we update.

Would it be easier to maintain the file without dates, or only dates for previous releases?

Comment (Lebbitt) [reply Changelog.md]: agreed - done

**Review by g-duff — APPROVED** [review]

**Pull request merged**